### DISCUSSION TITLE
How to Monitor Query Performance for Dashboards

### DISCUSSION THREAD

**Discussion by Asturias-sam** [apache/superset#13044] [General]
Is there any way in which we can measure the query performance for Dashboards, like in SQL LAB or Charts we get the execution time similar things we want for dashboard so it will help us the monitor the slowness for any dashboards ?

I am seeing this thing in logs:
<img width="1178" alt="Screenshot 2021-02-10 at 4 05 52 PM" src="https://user-images.githubusercontent.com/6216066/107499838-758a8200-6bbb-11eb-88f3-410b0cd8ddce.png">
What's the Duration here ?

**Comment by kamalkeshavani-aiinside** [comment]
As per my understanding, this action of 'dashboard' is just to load the UI of dashboard. To measure the performance of a dashboard, you would have to check the load time of different charts added in Dashboard.
One suggestion, you can consider the maximum value of a chart load time(inside the dashboard) as total dashboard load time.
